package.json

```diff
@@ -25,11 +25,13 @@
     "generate": "turbo run generate",
     "lint": "turbo run lint",
     "release": "turbo run build --filter=js... && turbo run build --filter=n8n-node... && changeset publish",
-    "test": "turbo run test",
+    "test": "playwright test",
```

Comment:
I don't think this is a good idea since we are also currently building jest unit tests that should run with `pnpm test` and we have (in theory) also `@formbricks/js` tests that run on `pnpm test`.
Is this used by playwright or do we trigger tests in the action differently? I would prefer to have another script like `test:e2e` or `playwright` for this.